src/iris/bin/sender.py

```diff
@@ -1145,7 +1150,7 @@ def fetch_and_send_message(send_queue, vendor_manager):
     except Exception:
         logger.exception('Failed to send message: %s', message)
         add_mode_stat(message['mode'], None)
-        if message['mode'] == 'email':
+        if message['mode'] == 'email' or message['role'] == 'literal_target':
```

Comment:
Switch to `unexpanded` flag as noted.

Reply:
This may also increment task failure if users give bad destinations, which we need to differentiate from actual task failures.